### PR TITLE
Add the talks

### PR DESCRIPTION
**Pull request by flobb**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,2 +1,393 @@
...
+    submarine or a cave and how to improve online performances with
+    local caching. The conference will be the occasion for the release
+    of a Gears-Admin generator plugin.
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/oncletom/an-offline-admin-generator-with-html-5-and-gears)  
+~~Video~~
+
+By Thomas Parisot
```
Comment (mykiwi) [inline readme.md:89]: https://twitter.com/oncletom
https://github.com/oncletom

File: readme.md [hunk 2]
```diff
@@ -1,2 +1,393 @@
...
+  <dt>Description</dt>
+  <dd>
+    Internationalization is an important feature in a worldwide project.
+    All-important projects required at some point to be used by people
+    with different cultures and languages. This constraint must be
+    understood from the application architecture to the implementation.
+  </dd>
+</dl>
+
+~~Slides~~
```
Comment (mykiwi) [inline readme.md]: http://slideshare.net/th0masr/internationalization-with-the-symfony-framework

File: readme.md [hunk 3]
```diff
@@ -1,2 +1,393 @@
...
+![twitter](icon/twitter.png) [@th0masr](https://twitter.com/th0masr)
+
+---
+
+## Cloud and evolution of the Microsoft platform
+
+~~Slides~~  
+~~Video~~
+
+By [Pierre Couzy](https://connect.sensiolabs.com/profile/piercou)
```
Comment (mykiwi) [inline readme.md]: https://twitter.com/piercou
https://github.com/piercou

File: readme.md [hunk 4]
```diff
@@ -1,2 +1,393 @@
...
+  <dt>Description</dt>
+  <dd>
+    Have you ever wondered what happens to an HTTP request when it
+    reaches a symfony application ? We are going to dive into the core
+    of symfony, watch what's happening under the hood and explain when
+    and how you can hookup to customize symfony to your needs.
+  </dd>
+</dl>
+
+~~Slides~~
```
Comment (mykiwi) [inline readme.md]: https://speakerdeck.com/ubermuda/symfonys-interna

File: readme.md [hunk 5]
```diff
@@ -1,2 +1,393 @@
...
+  <dd>
+    Jonathan will present on the latest version of Doctrine, 2.0. As the
+    title suggests, Doctrine 2 is not the same old PHP ORM. It has been
+    almost completely re-designed and coded from the ground up. The new
+    version addresses performance problems, limitations and design
+    problems of Doctrine 1.
+  </dd>
+</dl>
+
+~~Slides~~
```
Comment (mykiwi) [inline readme.md]: http://slideshare.net/jwage/doctrine-2-not-the-same-old-php-orm

File: readme.md [hunk 6]
```diff
@@ -1,2 +1,393 @@
...
+    your application jump started and well under way within minutes, but
+    what happens if you need something outside that core? What if you see
+    something in Zend Framework you wish to use?
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/weierophinney/using-zend-framework-with-symfony)  
+~~Video~~
+
+By Matthew Weier O'Phinney
```
Comment (mykiwi) [inline readme.md:253]: https://github.com/weierophinney

File: readme.md [hunk 7]
```diff
@@ -1,2 +1,393 @@
...
+    We will cover the technical underpinnings of the system, common tips
+    and best practices for getting the most out of Git and ways to use Git
+    that you may not have thought of before – as a pure
+    content-addressable filesystem. Attendees will leave with a better
+    understanding of Git and hopefully a new way of thinking about how to
+    work.
+  </dd>
+</dl>
+
+~~Slides~~
```
Comment (mykiwi) [inline readme.md]: https://speakerdeck.com/schacon/introduction-to-git (I guess it's the same)

File: readme.md [hunk 8]
```diff
@@ -1,2 +1,393 @@
...
+  <dt>Description</dt>
+  <dd>
+    In this session we'll talk about implementing a Symfony-based CMS
+    system in an editorial company. Will be based on our experience
+    building web CMS solutions for traditional publishing companies
+    and will cover all steps on this kind of projects.
+  </dd>
+</dl>
+
+~~Slides~~
```
Comment (mykiwi) [inline readme.md]: http://slideshare.net/esmiz/implementing-a-symfony-based-cms-in-a-publishing-company

File: readme.md [hunk 9]
```diff
@@ -1,2 +1,393 @@
...
+  <dt>Description</dt>
+  <dd>
+    I've been talking about Symfony for 2 years. It's time to actually
+    reveal the Symfony 2 MVC framework. I will show you how to use
+    Symfony 2, and this session will also be the occasion to publish
+    the first alpha release Symfony 2. Don't miss this session!
+  </dd>
+</dl>
+
+~~Slides~~
```
Comment (mykiwi) [inline readme.md]: http://slideshare.net/fabpot/symfony-20-revealed

File: readme.md (hunk 1 above)
Comment (flobb) [inline readme.md:89]: Done

File: readme.md (hunk 6 above)
Comment (flobb) [inline readme.md:253]: Done

File: readme.md [hunk 10]
```diff
@@ -1,2 +1,398 @@
 # [Symfony Live - Paris 2010](http://paris2010.live.symfony.com/) talks
 
+- All talks are in **english**
```
Comment (romqin) [inline readme.md]: `**french** or **english**`

**Comment by mykiwi** [comment]
:+1: Thanks @florianbelhomme

**Pull request merged**